### PR TITLE
Change prints to logs

### PR DESCRIPTION
**Pull request by LipuFei**
Fix issue #51

### CODE REVIEW

**Comment by LipuFei** [comment]
@rauljim ready to be reviewed and merged.

File: core/bootstrap.py [hunk 1]
```diff
@@ -113,21 +113,21 @@ def get_sample_unstable_addrs(self, num_addrs):
             i_warn_you_msg = "You are messing with my off-line detector, my friend"
             logger.warning(i_warn_you_msg)
         if 1:#len(self._unstable_ip_port) < num_addrs:
-            print '>>>len(self._unstable_ip_port) == %d, num_addrs: %d' % (
-                len(self._unstable_ip_port), num_addrs)
-            
+            logger.debug('>>>len(self._unstable_ip_port) == %d, num_addrs: %d' % (
+                len(self._unstable_ip_port), num_addrs))
```
Comment (rauljim) [inline core/bootstrap.py]: Have you tested this?

It seems to me that this line will trigger same problem as https://github.com/rauljim/pymdht/pull/50

**Comment by rauljim** [comment]
Overall, it looks good to me. See inline comment.
Just make sure it passes unittests. Have you run nosetests on pymdht?

File: core/bootstrap.py (hunk 1 above)
Comment (LipuFei) [inline core/bootstrap.py]: @rauljim These are just numbers, they should be fine. I will review and fix the others.

**Comment by LipuFei** [comment]
@rauljim I've run the tests. Nothing wrong with the logging stuff.

Two tests failed. I tested with some earily versions and they still failed, so I guess they were broken long ago. I will create an issue for this.

**Comment by LipuFei** [comment]
@rauljim this one is ready to be merged. I've created an issue for the failing unit tests #53

**Comment by rauljim** [comment]
Good. I merge.

**Pull request merged**